cvat/settings/testing.py

```diff
@@ -64,6 +64,11 @@
     'django.contrib.auth.hashers.MD5PasswordHasher',
 )
 
+# Most of the tests don't suppose user limitations
+# When needed, this can be enabled with a decorator
+# and tested specifically
+IAM_USER_LIMITS_ENABLED = False
```

Comment:
I don't like the approach. It is much better to have a configuration with default values for limits. Another approach is to use a function to update limits directly in old tests. For example, during login, the limits can be updated for a user.